ci/do_ci.sh

```diff
@@ -8,22 +8,21 @@ set -e
 echo "building using ${NUM_CPUS} CPUs"
 
 function bazel_release_binary_build() {
-  cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building..."
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt @envoy//source/exe:envoy-static.stripped.stamped
+  cd "${ENVOY_BUILD_DIR}"
```

Comment:
Do we need this? Shouldn't we build from within source (current) directory? Otherwise, this results in `Modified` git version, since none of the source files are in the tree.